src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -196,6 +196,33 @@ public Task Format()
             return Task.CompletedTask;
         }
 
+        [RavenAction("/admin/debug/storage/environment/report", "GET", AuthorizationStatus.ClusterAdmin)]
```

Comment:
I think this should be `Operator`, is there a reason this is cluster admin level?

Reply:
Need to be moved to StorageHandler

Reply:
done

Reply:
done